tests/data/scripts_pythondistdeps/test-data.yaml

```diff
@@ -1214,3 +1214,49 @@
                 python3dist(backports-range) = 3.7.2
                 python3dist(backports.range) = 3.7.2
             requires: python(abi) = 3.7
+--requires --normalized-names-format pep503 --package-name +docs:
```

Comment:
why is the package name `+docs` and how does the generator know that?

Reply:
To be discussed on BJ.

Reply:
Done